### PR TITLE
add config for test suite

### PR DESCRIPTION
**Pull request by adeelahmad**
Closes AMS2-72

### CODE REVIEW

**Comment by afred** [comment]
Looking into why this is failing.

**Comment by adeelahmad** [comment]
@afred it should be ready to merge now.

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +37,6 @@ deploy:
   region: "us-east-1"
   on:
     branch: develop
-deploy:
   provider: codedeploy
```
[the remark this answers is not in the document]
Comment (adeelahmad) [reply .travis.yml]: @afred We just need to list multiple providers under deploy as per travis docs: https://docs.travis-ci.com/user/deployment#Deploying-to-Multiple-Providers

[the remark this answers is not in the document]
Comment (afred) [reply .travis.yml]: @adeelahmad Right, but the Travis docs indicate an array of hashes under the `deploy` key. The way this `.travis.yml` is set up, it's not an array of hashes, it's just a single hash.

**Comment by adeelahmad** [comment]
@afred this PR is ready to be reviewed again.

**Pull request merged**